javascript/extractor/src/com/semmle/js/parser/TypeScriptParser.java

```diff
@@ -236,11 +251,16 @@ private void setupParserWrapper() {
     int reserveMemoryMb = getMegabyteCountFromPrefixedEnv(TYPESCRIPT_RAM_RESERVE_SUFFIX, 400);
 
     File parserWrapper = getParserWrapper();
-    
-    List<String> cmd = getNodeJsRuntimeInvocation(
-        "--max_old_space_size=" + (mainMemoryMb + reserveMemoryMb),
-        parserWrapper.getAbsolutePath()
-    );
+
+    String debugFlagString = Env.systemEnv().getNonEmpty(TYPESCRIPT_NODE_FLAGS);
+    List<String> debugFlags = debugFlagString == null ? new ArrayList<>() : debugFlagString.split(" ");
+    debugFlags.retainAll(allowedDebugNodeFlags);
```

Comment:
I agree that we should avoid throwing an exception when unsupported flags are supplied, but
could we at least print a warning about the use of an unsupported flag, such that we don't just silently ignore them?